utils.go

```diff
@@ -280,6 +280,11 @@ var (
 	}
 )
 
+var (
+	validKafkaVersion1 = regexp.MustCompile(`^0\.\d+\.\d+\.\d+$`)
```

Comment:
Thanks for raising this, I think we could probably be a bit more clear with the variable names here and probably should just add a comment
```suggestion
        //This regex validates that a string complies with the pre kafka 1.0.0 format for version strings, for example 0.11.0.3
	validPreKafka1Version = regexp.MustCompile(`^0\.\d+\.\d+\.\d+$`)
```